ffc/classname.py

```diff
@@ -6,6 +6,11 @@
 # SPDX-License-Identifier:    LGPL-3.0-or-later
 """Some basics for generating C code."""
 
+import ffc
+import ufl
+import hashlib
+from ffc.parameters import compute_jit_parameters_signature
```

Comment:
Could `compute_jit_parameters_signature` be just `compute_jit_signature` since it's already namespaced by the module  `parameters`?

Reply:
👍